sdk/communication/communication-call-automation/src/callMedia.ts

```diff
@@ -173,4 +176,63 @@ export class CallMedia {
   public async cancelAllOperations(): Promise<void> {
     return this.callMedia.cancelAllMediaOperations(this.callConnectionId, {});
   }
+
+  /**
+   * Start continuous Dtmf recognition by subscribing to tones.
+   * @param targetParticipant - Target participant.
+   * @param operationContext - The value to identify context of the operation.
+   * */
+  public async startContinuousDtmfRecognition(
+    targetParticipant: CommunicationIdentifier,
+    operationContext?: string
```

Comment:
Please create an interface that extends OperationOptions in models/options.ts to hold all optional parameters in this method. For example:
public async startRecognizing(
    targetParticipant: CommunicationIdentifier,
    maxTonesToCollect: number,
    **recognizeOptions: CallMediaRecognizeDtmfOptions**
  ): Promise<void>